README.md

```diff
@@ -14,14 +14,18 @@ This module allows users to create and manage keys, key rings, and key policies
 
 ## Table of Contents
 
-1. [Usage](#usage)
-1. [Examples](#examples)
-1. [Modules](#modules)
-1. [Key Management Instance Types](#key-management-instance-types)
-1. [Resources](#resources)
-1. [Inputs](#inputs)
-1. [Outputs](#outputs)
-1. [Contributing](#contributing)
+- [Terraform IBM ICSE Key Management Module](#terraform-ibm-icse-key-management-module)
+  - [Table of Contents](#table-of-contents)
```

Comment:
looks like a mistake?

Reply:
This happened implicitly during pre-commit hooks runs. Reverted the changes.